src/unoserver/converter.py

```diff
@@ -135,6 +136,9 @@ def convert(self, inpath=None, indata=None, outpath=None, convert_to=None):
             # TODO: Verify that inpath exists and is openable, and that outdir exists, because uno's
             # exceptions are completely useless!
 
+            if not Path(inpath).exists():
+                raise RuntimeError(f"Opening path {inpath} not exists.")
```

Comment:
nit: I'd say either `f"Opening path {inpath} failed"` or `f"Path {inpath} does not exist"` - the current phrase is a mixture of both ;)

Reply:
Changed!